src/Toolkit/Toolkit.UI.Controls/MeasureToolbar/MeasureToolbar.cs

```diff
@@ -631,26 +625,7 @@ private static void OnLineSketchEditorPropertyChanged(DependencyObject d, Depend
         /// Identifies the <see cref="AreaSketchEditor"/> dependency property.
         /// </summary>
         public static readonly DependencyProperty AreaSketchEditorProperty =
-            DependencyProperty.Register(nameof(AreaSketchEditor), typeof(SketchEditor), typeof(MeasureToolbar), new PropertyMetadata(null, OnLineSketchEditorPropertyChanged));
```

Comment:
This is the only bug. It should call  OnAreaSketchEditorPropertyChanged instead, and don't delete the code below.